pkg/virtualization/core/storage/disk/virtualharddisksettingdata.go

```diff
@@ -74,3 +91,54 @@ func GetVirtualHardDiskSettingData(whost *host.WmiHost, path string,
 
 	return
 }
+
+func GetVirtualHardDiskSettingDataFromXml(whost *host.WmiHost, xmlInstance string) (size uint64,
+	blockSize uint32,
+	lSectorSize uint32,
+	pSectorSize uint32,
+	format uint16,
```

Comment:
Do we use these in gRPC call? Just let me know that protobuf doesn't support unit16, so it would be better to use unit32 (https://protobuf.dev/reference/protobuf/proto3-spec/)

Reply:
That is a good catch. I have used the uint16 data type to match the WMI structure - https://learn.microsoft.com/en-us/windows/win32/hyperv_v2/msvm-virtualharddisksettingdata

That being said, this data type is not used directly in gRPC call and converted in another data type in function - https://github.com/microsoft/moc-pkg/pull/481/files#diff-b34b7f48298efd044258ebc67c8fa4772b36c3b9e8103eca1dcaef57313cb31c